### ISSUE TITLE
Running Tutorial in colab produces consistently zero evaluation Average precision while training

### ISSUE THREAD

**Issue by RoozbehSanaei** [google/automl#684]

**Comment by kartik4949** [comment]
@RoozbehSanaei are you able to train the model? without having  Momentum variable not found 
issue?

**Comment by RoozbehSanaei** [comment]
I did'nt touch momentum variable and executed the notebook in colab just  as it is but evaluations are like

Accumulating evaluation results...
DONE (t=0.10s).
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.001
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000

**Comment by kartik4949** [comment]
> I did'nt touch momentum variable and executed the notebook in colab just as it is but evaluations are like
> 
> Accumulating evaluation results...
> DONE (t=0.10s).
> Average Precision (AP) @[ IoU=0.50:0.95 | area= all | maxDets=100 ] = 0.000
> Average Precision (AP) @[ IoU=0.50 | area= all | maxDets=100 ] = 0.000
> Average Precision (AP) @[ IoU=0.75 | area= all | maxDets=100 ] = 0.000
> Average Precision (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
> Average Precision (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
> Average Precision (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
> Average Recall (AR) @[ IoU=0.50:0.95 | area= all | maxDets= 1 ] = 0.000
> Average Recall (AR) @[ IoU=0.50:0.95 | area= all | maxDets= 10 ] = 0.000
> Average Recall (AR) @[ IoU=0.50:0.95 | area= all | maxDets=100 ] = 0.000
> Average Recall (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
> Average Recall (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.001
> Average Recall (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000

May be that notebook is not updated as per new PRs they have in this repo. @RoozbehSanaei

**Comment by mingxingtan** [comment]
Yeah, I can reproduce this with --val_json_file, but if you remove --val_json_file, then it works well (with non-zero AP)

Looks like the pascal json file has some issues.  I will try to debug it later. Please just remove --val_json_file for now.
